uint32[] number_of_neurons_
float64[] activity_

